collection-scripts/gather_network_logs

```diff
@@ -26,6 +26,46 @@ if [ "$NETWORK_TYPE" == "openshiftsdn" ]; then
         oc -n openshift-sdn exec $OVS_POD -- bash -c "cat /var/log/openvswitch/ovsdb-server.log" > ${NETWORK_LOG_PATH}/${OVS_POD}_ovsdb_log &
         PIDS+=($!)
     done
+elif [ "$NETWORK_TYPE" == "kuryr" ]; then
+      CONTROLLER_POD=$(oc -n openshift-kuryr get pods --no-headers -o custom-columns=':metadata.name' -l app=kuryr-controller | tail -1)
```

Comment:
Instead of ` --no-headers -o custom-columns=':metadata.name'` you can do `-o name`, but it'll get you `pod/` part too. Which still works with `oc logs` so it might be fine. No pushing here, whatever you prefer.

Reply:
Thanks! Your suggestion is indeed more clean, but it would break the creation of the log file.